src/domain/QuestionList/QuestionListHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import { ArrowLeftOutlined } from '@ant-design/icons';
+import { Button, Space } from 'antd';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import Header from '../../layout/header/Header';
+
+export default function QuestionListHeader({
+  isBeingEdited,
+  setBeingEditedCallback,
+  listTitle,
+  discardCallback,
+  saveChangesCallback,
+  setDrawerVisibilityCallback,
+}: {
+  isBeingEdited: boolean;
```

Comment:
I recommend move definition of props into separate file. It's hard to read code now.